### ISSUE TITLE
add-github does not work

### ISSUE THREAD

**Issue by A-noximous** [theRookieCoder/ferium#84]
Adding mods published on github through github releases with `ferium.exe add-github` does not work. Example:
`$ ./ferium.exe add-github Earthcomputer/clientcommands`
`Checking internet connection... ✓`
`Adding mod... The project/repository is not compatible`

[The mod](https://github.com/Earthcomputer/clientcommands) does publish jar files through releases.

Similar issue with [this mod](https://github.com/aleksilassila/litematica-printer).

All other mods from Modrinth and Curseforge work fine.

On a side note, the help text  in README.md for add-github is wrong, trying `add-github <owner> <name>` results in the following:
`$ ./ferium.exe add-github Earthcomputer clientcommands`
`error: Found argument 'clientcommands' which wasn't expected, or isn't valid in this context`

**Comment by theRookieCoder** [comment]
> The project/repository is not compatible

This is a different error. If Ferium doesn't find a jar file, it will error out with `The project/repository is not a mod`. I looked at the releases and the assets' file names don't specify the game version or mod loader, so you'll have to [disable these checks](https://github.com/theRookieCoder/ferium#check-overrides). The checks for GitHub do not work very well and there's not much I can do about this.

> help text in README.md for add-github is wrong

Yes I had recently changed it and it seems I haven't updated the readme. Will do right away!
